docs/examples/use_cases/tensorflow/yolov4/src/dali/ops.py

```diff
@@ -0,0 +1,212 @@
+import nvidia.dali as dali
+
+def input(file_root, annotations_file, device_id, num_threads, device, random_shuffle=True):
+    inputs, bboxes, classes = dali.fn.readers.coco(
+        file_root=file_root,
+        annotations_file=annotations_file,
+        ltrb=True,
+        shard_id=device_id,
+        num_shards=num_threads,
+        ratio=True,
+        random_shuffle=random_shuffle
+    )
+    images = dali.fn.decoders.image(inputs, device=device, output_type=dali.types.RGB)
+
+    return images, bboxes, classes
+
+def ltrb_to_xywh(bboxes):
+    Z = dali.types.Constant(0.0)
+    H = dali.types.Constant(0.5)
+    O = dali.types.Constant(1.0)
+    MO = dali.types.Constant(-1.0)
+
+    M = dali.fn.stack(
+            dali.fn.stack(H, Z, H, Z),
+            dali.fn.stack(Z, H, Z, H),
+            dali.fn.stack(MO, Z, O, Z),
+            dali.fn.stack(Z, MO, Z, O)
+        )
+    return dali.fn.coord_transform(bboxes, M=M)
```

Comment:
How about something like:
```suggestion
    return dali.fn.coord_transform(bboxes, M=[0.5, 0.0, 0.5, 0.0,
                                                                         0.0, 0.5, 0.0, 0.5,
                                                                         -1.0, 0.0, 1.0, 0.0,
                                                                         0.0, -1.0, 0.0, 1.0,])
```
Also I don't understand this 0.5 cooef.
isn't l==x and y==t, while w=r-l and h-b-t ?

Reply:
The xywh format used in darknet interprets xy as a coordinates of bbox center.

Reply:
I would add such comment in that case. In DALI xy is x,y of the corner, no the center.

Reply:
Refactored this function and added comment with coordinates format description.